invisible_cities/cities/components.py

```diff
@@ -211,10 +211,39 @@ def index_tables(file_out):
                 table.colinstances[colname].create_index()
 
 
+def flatten_dict(arg : dict,
```

Comment:
This name refers more to the solution you've had to implement for the case of nested dictionaries rather than the "problem" it solves. I suggest something along the lines of "stringify dict". Try to come up with a cool name :).